src/app/pages/server-details/server-details.component.scss

```
@@ -33,12 +33,12 @@
 :host ::ng-deep .block_half_taller {
   @apply px-6 py-6 bg-secondary border-emerald-400 border border-solid rounded-lg flex flex-col relative overflow-hidden;
   width: calc(50% - 1rem);
-  min-height: 615px;
-  height: 615px;
+  min-height: 530px;
+  // height: 530px;
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)